lib/mayaUsd/python/wrapPrimUpdater.cpp

```diff
@@ -246,6 +246,10 @@ void wrapPrimUpdaterContext()
             "GetArgs",
             &UsdMayaPrimUpdaterContext::GetArgs,
             boost::python::return_internal_reference<>())
+        .def(
+            "GetAdditionalFinalCommands",
+            &UsdMayaPrimUpdaterContext::GetAdditionalFinalCommands,
+            boost::python::return_internal_reference<>())
         .def("MapSdfPathToDagPath", &UsdMayaPrimUpdaterContext::MapSdfPathToDagPath);
```

Comment:
Unfortunately, calling this currently fails in Python because UFE does not expose the UndoableCommand and CompositeUndoableCommand to Python. Not a blocker until someone wants to add additional commands from aprim updater in Python.